.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
**Optional Changes**
- [ ] It is not good practice to commit `.vscode` settings to git/version control. Please let us add this file to `.gitignore` since it is a matter of user preference.